lib/erl_interface/test/ei_format_SUITE_data/ei_format_test.c

```diff
@@ -27,20 +27,27 @@
  * Author:  Jakob 
  */
 
-static void
+static int
```

Comment:
Maybe this change affects ei_connect somehow?
At this moment return type doesn't use anyhow - maybe return void back?

Reply:
That was a residue from an earlier attempt. I reverted it. An unused return value should be totally benign though.

Reply:
Looks like it fixed tests

Reply:
Correlation is not causation.
I think those tests are just a bit wobbly. I've seen them fail unrelated to this PR.